sycl/include/sycl/handler.hpp

```diff
@@ -1194,6 +1193,67 @@ class __SYCL_EXPORT handler {
       DidAdjust = true;
     };
 
+#ifdef __SYCL_EXP_PARALLEL_FOR_RANGE_ROUNDING__
```

Comment:
@intel/llvm-reviewers-runtime do I need to make this experimental using a compiler flag or do you think this should be default behaviour?